life/modernized/19-chapter-19.adoc

```diff
@@ -3101,6 +3181,10 @@ who had once professed with Friends, but was now become an opposer of them.
 This letter, containing advice which might advantageously apply to some in our times,
 is as follows:
 
+[.embedded-content-document.letter]
```

Comment:
So this wasn't embedding in the PDF in the modernized version for some reason. Original was fine.

Reply:
that's really weird... maybe it will embed when you put the -- back in above, where it was left out. I can't think of any other reason why it won't embed.

Reply:
Yes, it was the -- above....I just didn't look far enough up for it. I don't understand how I don't catch those when I compare the 2 PDF's side by side.....I mean that is EXACTLY the things I am looking for!! 😆